### ISSUE TITLE
Back button on Animal Details page

### ISSUE THREAD

**Issue by trevorskaggs** [trevorskaggs/shelterly#164]
"Animal Details page also needs a Back button, you get stuck here."

**Comment by AlexMountain** [comment]
Can you elaborate on this?  Where do they want to go from here?  Owner, SR, and Shelter should all show up if applicable and provide a link.  In addition, the sidebar links are available and every browser has a back button.  Did they get here from a search?  Should we be opening more details pages in a new tab?  Curious if that might solve things here.

**Comment by trevorskaggs** [comment]
I would agree, maybe opening those pages up as new tabs might solve a good part of the problem.  I'm a little concerned that older people might not "think in tabs" like we do, but if we start to make those "details" links into new tabs, that might help solve the problem in the short term.  Otherwise we are in the same problem as before that there really isnt a "back" in React, right?

**Comment by AlexMountain** [comment]
I built out list history tracking such that we can technically go "back" to wherever the user was previously, but there are use cases where it gets wonky, say, on a details page that was opened in a new tab - there would be no history to go back to.

I opened a ticket to make all clipboard details links from search results open in a new tab.  I had been meaning to make this change anyways.  If possible I'd like to try this first and get further feedback before implementing another back button here.
